src/Test.Utilities.Sarif/TestRuleBehaviors.cs

```diff
@@ -40,6 +41,8 @@ public enum TestRuleBehaviors
         // Assume one or more options are invalid
         RegardOptionsAsInvalid = 0x4000,
 
-        RaiseExceptionProcessingBaseline = 0x8000
+        RaiseExceptionProcessingBaseline = 0x8000,
+
+        RaiseExceptionProcessingPostUri = 0x10000
```

Comment:
>RaiseExceptionProcessingPostUri [](http://example.com/codeflow?start=8&length=31)

We don't 'process' a URI, we are posting the log contents. Or we are publishing it.

RaiseExceptionPostingLogFile